astropy/vo/client/vos_catalog.py

```diff
@@ -543,7 +539,7 @@ def from_registry(cls, registry_url, timeout=60, **kwargs):
 
         """
         # Download registry as VO table
-        with REMOTE_TIMEOUT.set_temp(timeout):
+        with conf.remote_timeout.set_temp(timeout):
```

Comment:
This is either incorrect or confusing. This `conf` is most likely the one imported in Line 20, which is from `io.votable`, not `utils.data`. Also at some other places in the same module, `conf` is overwritten by `from .. import conf`. I think the `conf` import lines need some clean up.